pkg/3scale/amp/operator/redis_options_provider.go

```diff
@@ -242,3 +243,9 @@ func (r *RedisOptionsProvider) backendRedisPodTemplateLabels() map[string]string
 
 	return labels
 }
+
+func (r *RedisOptionsProvider) setPriorityClassNames() {
```

Comment:
Missing backend redis priority class name ?

Reply:
Added. Thank you